core-web/apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-create-edit/dot-template-props/dot-template-props.component.html

```diff
@@ -1,12 +1,12 @@
 <dot-form-dialog
-    data-testId="dialogForm"
     [saveButtonDisabled]="(isFormValid$ | async) === false"
     (cancel)="onCancel()"
     (save)="onSave()"
+    data-testId="dialogForm"
 >
-    <form class="p-fluid" [formGroup]="form" data-testId="form" *ngIf="form">
+    <form class="p-fluid" *ngIf="form" [formGroup]="form" data-testId="form">
         <div class="field" data-testId="titleField">
-            <label for="title" class="p-label-input-required">{{
+            <label for="title" dotFieldRequired>{{
```

Comment:
missing for attribute here.

Reply:
it is for field and label not for directive.